core/src/main/java/com/google/instrumentation/stats/StatsContext.java

```diff
@@ -57,7 +57,7 @@ public final StatsContext with(
    *
    * @param output the {@link OutputStream} to add the serialized form of this {@link StatsContext}.
    */
-  public abstract void serialize(OutputStream output) throws IOException;
+  public abstract void serialize(OutputStream output) throws ParseException;
```

Comment:
Yeah that also makes sense. I'll update `serialize` to throw `IOException` as before.